straxen/plugins/event_processing.py

```diff
@@ -522,27 +522,50 @@ class EventPositions(strax.Plugin):
                 '?version=ONLINE&run_id=plugin.run_id',
         help='Electron drift time from the gate in ns',
         cache=True)
+    
+    def infer_dtype(self):
+        dtype = []
+        for j in 'x y r'.split():
```

Comment:
It is indeed trivial, but on the other hand this is rather used and it is something I guess most of the people would define by themselves if it was not available, leading anyway to the same amount of used RAM. From the storage point of view this could help, but being anyway just at the even level I do not think it is a real issue

Reply:
Okay fine with.